wiki/Hit_Objects/Slider/en.md

```diff
@@ -0,0 +1,33 @@
+
+Slider ticks are small circles that appear in regular intervals along the sliderbody. The player only fails to completely clear a slider if the cursor is outside the slider ball range (defined by the circle around the ball) or the mouse button is unpressed while the slider ball is passing over a slider tick. Being outside the slider ball range or unpressing the button at any other time will not yield any penalty (though it is not advised as it could easily result in a sliderbreak). Passing over a sliderend or a slider tick will give a small health boost; usually just barely enough to keep the health bar where it is.
+
+## Editor
+
+Sliders can be created in the [beatmap editor](/wiki/Beatmap_Editor) by selecting the Slider icon from the selection menu on the left, and clicking once on the playfield. This will create a sliderhead in the selected position. Moving the cursor will show a preview of how the slider will look if you finish it in the current spot. Slideranchors can now be placed to manipulate the path of the sliderbody. To curve a slider, place a slideranchor by left-clicking anywhere on the playfield. To create sharp angles double-click instead. This will create a red slideranchor. Once you're satisfied with the placement of your slider, right-click to end it.
+
+You are free to move/add/remove slider control points after the slider is finished. You can remove control points by right clicking them after the slider has been selected, or add a control point by holding `Ctrl` and clicking anywhere inside the sliders (double click or control+click a slider point to make a red control point). Sliders can also be reversed by clicking and dragging the sliderend in the timeline.
+
+Users can also adjust the number of ticks on their beatmap's sliders in the `Timing` section. More ticks make the combo rise faster (1x more per tick) and thus offer the opportunity for higher scores, but at the same time they provide more chances for the player to fail as judgement becomes harsher.
+
+## Hitsounding Sliders
+
+[Hitsounding](/wiki/Glossary) sliders is most effectively done by selecting each part of a slider by double-clicking it in the [timeline](/wiki/Beatmap_Editor/Timelines). This will highlight the element in red and limit hitsounds to the selected part. If this selection isn't done, hitsounds will apply to the whole slider and will sound confusing for sliders that cover multiple, different sounds.
```

Comment:
I'm not sure if this would be the right place to put it, but I've had this note on a clipboard for a long ass time (said in december 2018 apparently)

> [9:33 PM] UndeadCapulet: nop kuro’s right, you need to change the sampleset if you want to change the sliderslide volume mid-slider

seems important to note *somewhere* but maybe it's such a technical detail it belongs on .osu file format or something. idk

Reply:
think it's fine here. adding it elsewhere would also be good though